source/Octopus.Tentacle/Scripts/ScriptWorkspace.cs

```diff
@@ -82,11 +82,6 @@ public string ResolvePath(string fileName)
             return path;
         }
 
-        public void Delete()
```

Comment:
Removing Sync code path that does a Wait on an Async call as we can be fully async now

Reply:
Leaving FileSystem.DeleteDirectory sync method as it is used in other places in the tests. Future cleanup task.